src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -94,18 +96,22 @@ public DatadogBuildListener() { }
   @Override
   public final void onStarted(final Run run, final TaskListener listener) {
     String jobName = run.getParent().getFullName();
-    logger.fine(String.format("onStarted() called with jobName: %s", jobName));
-    HashMap<String,String> tags = new HashMap<String,String>();
+    Map<String,String> tags = new HashMap<String,String>();
 
     // Process only if job is NOT in blacklist and is in whitelist
     if ( DatadogUtilities.isJobTracked(jobName) ) {
       logger.fine("Started build!");
+
+      Boolean useJobRegex = DatadogUtilities.getDatadogDescriptor().getJobRegex();
```

Comment:
Change Boolean to its primitive. This will avoid creating a new object. Also, do we have to check for null here?

Reply:
After talking with @sjenriquez about the scope/UI complexity here we decided to move to a new text area for the tags option. In doing this I removed the need for this variable entirely. Good to know though!